packages/js/src/server/integrations/unhandled_rejection_plugin.ts

```diff
@@ -0,0 +1,17 @@
+import { Types } from '@honeybadger-io/core'
+import Client from '../../server'
+import UnhandledRejectionMonitor from './unhandled_rejection_monitor'
+
+export default function (): Types.Plugin {
+  return {
+    load: (client: typeof Client) => {
+      if (!client.config.enableUnhandledRejection) {
+        return
+      }
+      const unhandledRejectionMonitor = new UnhandledRejectionMonitor(client)
+      process.on('unhandledRejection', function honeybadgerUnhandledRejectionListener(reason, _promise) {
```

Comment:
FYI: We usually prefix a parameter with `_` so that eslint won't pick it up and say that _parameter is not used_ (https://github.com/honeybadger-io/honeybadger-js/blob/master/.eslintrc#L33).

Reply:
I tried to make as little change as possible to the existing code with this PR. In the existing code, the param was called `_promise` and was unused, so I just left it that way. 

I guess technically this one is now "used" in that it's passed through... but it's passed through to a function that doesn't use it, so it seemed more consistent to keep the naming the same throughout.

Reply:
OK, thanks for the details. We can remove this on the next iteration, to avoid the confusion in the future.